app/views/org_admin/templates/_templates_list.html.erb

```diff
@@ -0,0 +1,66 @@
+<% # locals: templates, current_org %>
+
+<div class="table-responsive">
+  <table class="table table-hover table-bordered tablesorter">
+    <thead>
+      <% if scope.length > TABLE_FILTER_MIN_ROWS %>
+        <tr>
+          <th colspan="5" class="sorter-false">
+            <%= render(partial: "shared/table_filter", locals: { placeholder: _('Search')}) %>
+          </th>
+        </tr>
+      <% end %>
+      <tr>
+        <th><%= _('Template Name') %></th>
+        <th><%= current_user.can_super_admin? ? _('Organisation') : _('Description') %></th>
+        <th><%= _('Status') %></th>
+        <th><%= _('Edited Date') %></th>
+        <th class="sorter-false"></th>
+      </tr>
+    </thead>
+    <tbody>
+    <% scope.each do |template| %>
+      <tr>
+        <td>
+          <%= template.title %>
+        </td>
+          <td>
+            <%= current_user.can_super_admin? ? template.org.name : raw(template.description) %>
+          </td>
+          <td>
+            <%= (template.published? ? _('Published') : _('Unpublished')) %>
```

Comment:
Published status shouldn't be binary.
This explains the bugs @stephaniesimms highlighted in the meeting yesterday with templates appearing unpublished once you edit them.
Have a look at the logic that was in place before which correctly displays the status:
```
<% if hash[:live].nil? %>
    <%= _('Unpublished') %>
<% elsif hash[:current].dirty? %>
    <%= _('Unpublished changes') %>
<% else %>
    <%= _('Published') %>
<% end %>
```

Reply:
Ah right. will address this and move that ticket over to review